manifest.json

```diff
@@ -0,0 +1,144 @@
+{
+  "dependencies": {
+    "angular": {
+      "version": "1.4.7",
+      "path": "https://ajax.googleapis.com/ajax/libs/angularjs/1.4.7",
+      "dest_target": "angularjs-",
```

Comment:
Here and below: in some places you use _, in others you use camelCase. We should be consistent -- probably camelCase.